convert2rhel/checks.py

```diff
@@ -698,3 +699,21 @@ def is_loaded_kernel_latest():
                 "If you wish to ignore this message, set the environment variable "
                 "'CONVERT2RHEL_UNSUPPORTED_SKIP_KERNEL_CURRENCY_CHECK' to 1." % package_to_check
             )
+
+
+def check_dbus_is_running():
+    """Error out if we need to register with rhsm and the dbus daemon is not running."""
+    logger.task("Prepare: Check that DBus Daemon is running")
+
+    if tool_opts.no_rhsm:
+        logger.info("Skipping the check because we have been asked not to subscribe this system to rhsm.")
+        return None
+
+    if not system_info.dbus_running:
+        logger.critical(
+            "Could not find a running DBus Daemon which is needed to register with subscription manager.\n"
+            "Please start dbus using `systemctl start dbus` or (on CentOS 6), `service messagebus start`"
+        )
+
+    logger.info("DBus Daemon is running")
+    return None
```

Comment:
minor: only `return` is enough, or even no `return` is needed here

Reply:
I was a bit on the fence about this one.  I kept the `None`s in because I am returning in two different levels but the counter argument is that we aren't using the return value from any of the `check_*` functions.  I've restructured this code so that the returns are both "exit early" returns at the top.  Using bare `return`s with that structure feel much better to me.